src/components/Countdown.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react'
+
+function Countdown({ Date }) {
```

Comment:
parameters preferred to name using camelCase notation.
Use PascalCase to name Components, or pass Components as props